_installation/single-multiple.md

```diff
@@ -10,13 +10,20 @@ Babelfish offers two modes of operation:
 - `single-db`
 - `multi-db`
 
-What is the purpose of these two modes? In MS SQL, it is really cheap to perform
-joins between tables located in different databases. In PostgreSQL, it is also
-possible to join between databases, but it is far more expensive to do so. Since
-joining different databases is pretty common in MS SQL, this is an important
-issue. The question now is: How can Babelfish model the desired MS SQL behavior?
-In the case of `single-db`, MS SQL databases are mapped to schemas in PostgreSQL. In
-the case of `multi-db`, many databases are used instead of just one.
+When you create a Babelfish cluster, you choose between using a single
+SQL Server database on its own or multiple SQL Server databases together.
```

Comment:
change "using" to "migrating"

Reply:
You misunderstood the text, which indicates that it needs improvement.
I am talking about the thing that is created when you connect via TDS and say `CREATE DATABASE`.
I had named it "SQL Server database" to distinguish it from a PostgreSQL database...
Lacking a better idea, I have renamed them to "T-SQL database" and "T-SQL schema".

Reply:
The correct terminology is "SQL Server database". Probably better is "migrated SQL Server database"

Reply:
So, whenever you want to refer to the thing that is created by `CREATE DATABASE` in Babelfish, you would call it a "migrated SQL Server database", even if it was not migrated at all?